visualization/app/codeCharta/ui/resetSettingsButton/resetSettingsButton.component.html

```diff
@@ -1 +1,3 @@
-<md-button class="md-primary" ng-click="$ctrl.applyDefaultSettings()"><i class="fa fa-undo"></i></md-button>
+<md-button class="md-primary" ng-click="$ctrl.applyDefaultSettings()"
+	><i class="fa fa-undo" title="Reset settings in this panel"></i
```

Comment:
```suggestion
	><i class="fa fa-undo" title="Reset settings to their defaults"></i
```

Reply:
Adressed with the individual reset settings button tooltips